ci-builder/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This Dockerfile is used in openshift CI
 FROM quay.io/fedora/fedora:40
```

Comment:
While you bump this, it might make also sense to bump to `fedora:41` (or possibly `latest`).